README.md

```diff
@@ -175,30 +175,47 @@ expect(1..10).to cover(3)
 ### Collection membership
 
 ```ruby
-expect(actual).to include(expected)
+# exact order, entire range
+expect(actual).to eq(expected)
+
+# exact order, partial range (based on an exact position)
 expect(actual).to start_with(expected)
 expect(actual).to end_with(expected)
 
-expect(actual).to contain_exactly(individual, items)
-# ...which is the same as:
-expect(actual).to match_array(expected_array)
+# any order, entire range
+expect(actual).to match_array(expected)
+
+# Assuming that:
+#   expected = [expected_element1, expected_element2]
```

Comment:
This comment is confusing.  Initially I read it to be saying, "this line assumes the array is in the exact order `[expected_element1, expected_element2]` but it does not assume any order.  Now I'm realizing that `expected` here refers to `expected` in the `expect(actual).to match_array(expected)` line above.  But that was totally not obvious and took me reading this a few times to realize.

I don't think these 2 lines are necessary and I think they confuse more than they clarify.